frappe/model/document.py

```diff
@@ -183,6 +183,21 @@ def load_from_db(self):
 				or []
 			)
 
+			if self.doctype not in core_doctypes_list:
+				for ch_df in frappe.get_meta(df.options).get_table_fields():
+					for ch in children:
+						ch[ch_df.fieldname] = (
+							frappe.db.get_values(
+								ch_df.options,
+								{"parent": ch.name, "parenttype": ch_df.parent, "parentfield": ch_df.fieldname},
+								"*",
+								as_dict=True,
+								order_by="idx asc",
+								for_update=self.flags.for_update,
+							)
+							or []
+						)
```

Comment:
This block handles(/fetches) multi-select fields inside a child table (row) if any

Reply:
However, the get_doc APIs don't allow for saving/inserting something like that yet (fails to insert data silently) - although the system allows having this.